### PR TITLE
Fix a bug the remote control view is not shown after PIP mode

### PR DESCRIPTION
**Pull request by jdragonbae**
This patch fixes a bug the remote control view is not shown after PIP
mode. Instead of using pip controller, it works same as home button
clicked.

Refs: #620

### CODE REVIEW

**Comment by jdragonbae** [comment]
After this patch, there is an issue where the navigation bar does not disappear.

**Comment by romandev** [comment]
Did you try the approach to get the PIP controller member in ChromeActivity?

**Comment by jdragonbae** [comment]
> Did you try the approach to get the PIP controller member in ChromeActivity?

Yes. It seems to work as same.

File: upstream/tb_chrome_android_java_src_org_chromium_chrome_browser_ChromeActivity.java.patch [hunk 1]
```diff
@@ -13,7 +13,21 @@ diff --git tb/chrome/android/java/src/org/chromium/chrome/browser/ChromeActivity
...
              public void onCrash(Tab tab) {
                  postDeferredStartupIfNeeded();
-@@ -2024,9 +2029,13 @@ public abstract class ChromeActivity<C extends ChromeActivityComponent>
+@@ -924,6 +929,13 @@ public abstract class ChromeActivity<C extends ChromeActivityComponent>
+         getManualFillingComponent().onResume();
+     }
+ 
++    public PictureInPictureController getPipController() {
++        if (mPictureInPictureController == null) {
++            mPictureInPictureController = new PictureInPictureController();
```
Comment (romandev) [inline upstream/tb_chrome_android_java_src_org_chromium_chrome_browser_ChromeActivity.java.patch:22]: Do we have to create a new instance?

**Comment by jdragonbae** [comment]
I changed it to get the PIP controller in ChromeActivity.

File: banana_cake/java/org/banana/cake/CakePipController.java [hunk 2]
```diff
@@ -25,7 +25,14 @@
             mPipController = null;
             return;
         }
-        mPipController = new PictureInPictureController();
+
+        ChromeActivity activity = getLastTrackedFocusedChromeActivity();
+        if (activity == null) {
+            mPipController = null;
```
Comment (romandev) [inline banana_cake/java/org/banana/cake/CakePipController.java:31]: We don't need to keep this because this causes a memory leak.

Comment (jdragonbae) [reply banana_cake/java/org/banana/cake/CakePipController.java:31]: Ack.

File: upstream/tb_chrome_android_java_src_org_chromium_chrome_browser_ChromeActivity.java.patch (hunk 1 above)
Comment (jdragonbae) [reply upstream/tb_chrome_android_java_src_org_chromium_chrome_browser_ChromeActivity.java.patch:22]: We can just return `mPictureInPictureController`.

Comment (romandev) [reply upstream/tb_chrome_android_java_src_org_chromium_chrome_browser_ChromeActivity.java.patch:22]: Make sense!

**Review by romandev — APPROVED** [review]
LGTM
Just query: Does your patch fix the issue that the home button is shown when back to the fullscreen?

**Comment by jdragonbae** [comment]
> LGTM
> Just query: Does your patch fix the issue that the home button is shown when back to the fullscreen?

I'm trying to fix the issue you said!

**Pull request merged**